src/mscorlib/src/System/Threading/CancellationToken.cs

```diff
@@ -254,18 +188,8 @@ public CancellationTokenRegistration Register(Action callback, bool useSynchroni
         /// <returns>The <see cref="T:System.Threading.CancellationTokenRegistration"/> instance that can 
         /// be used to deregister the callback.</returns>
         /// <exception cref="T:System.ArgumentNullException"><paramref name="callback"/> is null.</exception>
-        public CancellationTokenRegistration Register(Action<Object> callback, Object state)
-        {
-            if (callback == null)
-                throw new ArgumentNullException(nameof(callback));
-
-            return Register(
-                callback,
-                state,
-                false, // useSync=false
-                true   // useExecutionContext=true
-             );
-        }
+        public CancellationTokenRegistration Register(Action<object> callback, object state) =>
+            Register(callback, state, useSyncContext: false, useExecutionContext: true);
```

Comment:
No throw on `null` callback? (Is with throw expression in `Register` above) - though above may not be needed.